services/director-v2/src/simcore_service_director_v2/models/domains/comp_tasks.py

```diff
@@ -18,6 +18,9 @@
 class ComputationTaskCreate(BaseModel):
```

Comment:
I really dislike the names of these class models! They are all bodies of requests, namely ``Create``, ``Stop`` etc ...
When I saw them in the handler function arguments i did not understand what was all about since in addition you called them there ``job``!! 

Moreover, these are schema models, and not domain, right?

Reply:
to be honnest, I still don't quite get the difference between the schema/domain stuff